libsolidity/ast/Types.cpp

```diff
@@ -413,6 +413,22 @@ u256 IntegerType::literalValue(Literal const* _literal) const
 	return u256(_literal->value());
 }
 
+bigint IntegerType::minValue() const
```

Comment:
Can you pull this change out into a separate PR? It could be used by the warning for var (and tight packing).